create_shards.c

```diff
@@ -236,14 +236,11 @@ master_create_worker_shards(PG_FUNCTION_ARGS)
 
 	for (shardIndex = 0; shardIndex < shardCount; shardIndex++)
 	{
-		uint64 shardId = NextSequenceId(SHARD_ID_SEQUENCE_NAME);
+		int64 shardId = -1;
```

Comment:
Not changing in this review, but I will go through and eradicate unsigned integers from `pg_shard` in another branch. (done)